pages/glossary/included-in-the-accessibility-tree.md

```diff
@@ -3,6 +3,8 @@ title: Included in the accessibility tree
 key: included-in-the-accessibility-tree
 unambiguous: true
 objective: true
+input_aspects:
```

Comment:
Accessibility tree is a subset of the DOM tree. Shouldn't we add DOM tree as well?

Reply:
I don't think so. The accessibility tree is considered given by whatever UA/AT we're considering (i.e. it is not the job of the rule or the tester to build it).